.devcontainer/post-create.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+
+function add_config_if_not_exist {
+    if ! grep -F -q "$1" $HOME/.bashrc; then
+        echo "$1" >> $HOME/.bashrc
+    fi
+}
```

Comment:
a little function that ensures that there's no duplication of lines in `~/.bashrc`

Reply:
Thanks